.gitignore

```diff
@@ -5,6 +5,7 @@ wandb_run.sh
 sweep_config.yaml
 .ml-job-preset.yml
 wandb
+algorithms/minari/testing.py
```

Comment:
local (too local) development pipeline should not be propagated into the gitignore

Reply:
will remove when PR will be ready for review